vipps-checkout-api.md

```diff
@@ -252,9 +252,9 @@ var vippsCheckout = VippsCheckout({
 this should be considered pilot functionality. While fully functional please bear
 this in mind as you integrate.
 
-With Vipps Checkout Elements you can adjust the fields and values present in the Checkout. For example: you might have a purchasing flow where you do not require an address because you are not sending physical goods, or you do not need the customer to identify itself because the customer is already logged into your system.
+With Vipps Checkout Elements, you can adjust the fields and values present in the Checkout. For example, you might have a purchasing flow where you do not require an address because you are not sending physical goods, or you do not need the customer to identify themself because they is already logged into your system.
```

Comment:
"they are" in the last part of the last sentence. Not "they is"

Reply:
Uh oh!!
Fixed!